zeebe/atomix/cluster/src/main/java/io/atomix/cluster/messaging/MessagingConfig.java

```diff
@@ -288,6 +290,24 @@ public MessagingConfig setSocketReceiveBuffer(final int socketReceiveBuffer) {
     return this;
   }
 
+  public Duration getHeartbeatTimeout() {
```

Comment:
I'd just do this in the constructor of `NettyMessagingService` because that's where they become `final` and it's also where we "know" what valid values are.

Reply:
Done in 09b5ce29418ca62fb5ea8ba16e121a5f6d280318